example/example_generated_test.go

```diff
@@ -0,0 +1,1447 @@
+	accountlink "github.com/stripe/stripe-go/v72/accountlink"
+	balancetransaction "github.com/stripe/stripe-go/v72/balancetransaction"
+	capability "github.com/stripe/stripe-go/v72/capability"
+	charge "github.com/stripe/stripe-go/v72/charge"
+	session "github.com/stripe/stripe-go/v72/checkout/session"
+	countryspec "github.com/stripe/stripe-go/v72/countryspec"
+	coupon "github.com/stripe/stripe-go/v72/coupon"
+	customer "github.com/stripe/stripe-go/v72/customer"
+	customerbalancetransaction "github.com/stripe/stripe-go/v72/customerbalancetransaction"
+	dispute "github.com/stripe/stripe-go/v72/dispute"
+	event "github.com/stripe/stripe-go/v72/event"
+	invoice "github.com/stripe/stripe-go/v72/invoice"
+	invoiceitem "github.com/stripe/stripe-go/v72/invoiceitem"
+	authorization "github.com/stripe/stripe-go/v72/issuing/authorization"
+	card "github.com/stripe/stripe-go/v72/issuing/card"
+	cardholder "github.com/stripe/stripe-go/v72/issuing/cardholder"
+	transaction "github.com/stripe/stripe-go/v72/issuing/transaction"
+	mandate "github.com/stripe/stripe-go/v72/mandate"
+	order "github.com/stripe/stripe-go/v72/order"
+	orderreturn "github.com/stripe/stripe-go/v72/orderreturn"
+	paymentintent "github.com/stripe/stripe-go/v72/paymentintent"
+	paymentmethod "github.com/stripe/stripe-go/v72/paymentmethod"
+	payout "github.com/stripe/stripe-go/v72/payout"
+	person "github.com/stripe/stripe-go/v72/person"
+	plan "github.com/stripe/stripe-go/v72/plan"
+	price "github.com/stripe/stripe-go/v72/price"
+	product "github.com/stripe/stripe-go/v72/product"
+	promotioncode "github.com/stripe/stripe-go/v72/promotioncode"
+	earlyfraudwarning "github.com/stripe/stripe-go/v72/radar/earlyfraudwarning"
+	refund "github.com/stripe/stripe-go/v72/refund"
+	reversal "github.com/stripe/stripe-go/v72/reversal"
+	review "github.com/stripe/stripe-go/v72/review"
+	setupattempt "github.com/stripe/stripe-go/v72/setupattempt"
+	setupintent "github.com/stripe/stripe-go/v72/setupintent"
+	scheduledqueryrun "github.com/stripe/stripe-go/v72/sigma/scheduledqueryrun"
+	sku "github.com/stripe/stripe-go/v72/sku"
+	source "github.com/stripe/stripe-go/v72/source"
+	taxid "github.com/stripe/stripe-go/v72/taxid"
+	taxrate "github.com/stripe/stripe-go/v72/taxrate"
+	reader "github.com/stripe/stripe-go/v72/terminal/reader"
+	_ "github.com/stripe/stripe-go/v72/testing"
+	topup "github.com/stripe/stripe-go/v72/topup"
+	transfer "github.com/stripe/stripe-go/v72/transfer"
+	usagerecord "github.com/stripe/stripe-go/v72/usagerecord"
+	webhookendpoint "github.com/stripe/stripe-go/v72/webhookendpoint"
+)
+
+func TestCustomerList1(t *testing.T) {
+	params := &stripe.CustomerListParams{}
+	params.Filters.AddFilter("limit", "", "3")
+	result := customer.List(params)
+	assert.NotNil(t, result)
+}
+
+func TestBalanceTransactionRetrieve1(t *testing.T) {
+	result, _ := balancetransaction.Get("txn_xxxxxxxxxxxxx", nil)
+	assert.NotNil(t, result)
+}
+
+func TestBalanceTransactionList1(t *testing.T) {
```

Comment:
Could we remove the `1` for test names that aren't duplicated?

Reply:
ok let me regenerate the suites